src/components/search/index.tsx

```diff
@@ -25,7 +25,9 @@ export default function Search({ label, onChange }: SearchProps) {
   return (
     <Downshift
       initialInputValue={query}
-      onInputValueChange={(inputValue) => setQuery(inputValue || '')}
+      onInputValueChange={(inputValue) =>
+        debounce(setQuery, 500)(inputValue || '')
```

Comment:
Could see the `useDebounce` hook in webshop. That is quite handy to use the debounce concept in React context

Reply:
Oh dang. I see both my examples are the same (taken from webshop and app). And didn't have an example of the useState semantics. But that is OK enough I think :D might as well just keep it consistent and use the same concept here. At some point also maybe even reusable 😱😱

Reply:
Used that hook instead and moved the debouncing to the `useAutocomplete` hook.

Reply:
Looks good!